server/bin/fxa-content-server.js

```diff
@@ -156,9 +156,8 @@ function makeApp() {
   // log any joi validation errors
   app.use((err, req, res, next) => {
     if (err && err.isJoi) {
-      logger.error({
+      logger.error('validation.error',{
```

Comment:
@sunakshi96 need a style fix, there should be a space between `,{`, such that `, {`